src/main/java/com/zontik/groshiky/service/UserService.java

```diff
@@ -1,36 +1,46 @@
 package com.zontik.groshiky.service;
 
+import com.zontik.groshiky.dao.IRoleDao;
 import com.zontik.groshiky.dao.IUserDao;
+import com.zontik.groshiky.model.Role;
+import com.zontik.groshiky.model.Roles;
 import com.zontik.groshiky.model.User;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import java.util.Collections;
 
 
 @Service
 @Transactional
-public class UserService implements IUserService{
+public class UserService implements IUserService {
 
+    private final PasswordEncoder PasswordEncoder;
     private final IUserDao userDao;
+    private final IRoleDao roleDao;
 
     @Autowired
-    public UserService(IUserDao userDao) {
+    public UserService(IUserDao userDao, IRoleDao roleDao, PasswordEncoder PasswordEncoder) {
         this.userDao = userDao;
+        this.roleDao = roleDao;
+        this.PasswordEncoder = PasswordEncoder;
     }
 
-    @Override
-    public void createUser(User user) {
-        userDao.createUser(user);
+    public User createUser(User user) {
+        Role role = roleDao.findRoleByName(Roles.ADMIN);
+        user.setRoles(Collections.singletonList(role));
+        user.setPassword(PasswordEncoder.encode(user.getPassword()));
+        return userDao.save(user);
     }
 
-    @Override
+
     public User findByLogin(String login) {
-       return userDao.findByLogin(login);
+        return userDao.findByLogin(login);
     }
 
-    @Override
+
     public User findUserById(Integer id) {
-        return userDao.findById(id);
+        return userDao.getOne(id);
```

Comment:
Better to use "findById(id)"
You can read why here: https://www.javacodemonk.com/difference-between-getone-and-findbyid-in-spring-data-jpa-3a96c3ff